### ISSUE TITLE
Abitilty to edit boot device

### ISSUE THREAD

**Issue by axinojolais** [maas/python-libmaas#122]
Hi,

Could the library grow the ability to change the boot device of a node ? As far as I can see, it'd need to implement the "blockdevice" / "blockdevice_set" objects.

For what it's worth, I'm currently doing that with :
`maas $MAAS_PROFILE block-device set-boot-disk $system_id $blockdev_id`

Thanks !

**Comment by blakerouse** [comment]
Yes first we need to get block device objects in `libmaas` once that is complete you would be able to do:

```
sdb = [
    disk
    for disk in machine.block_devices
    if disk.name == 'sdb'
][0]
machine.set_boot_disk(sdb)
```

**Comment by blakerouse** [comment]
This was fixed in 2dc0b03d18eb0391b02f1266e4188ee8962bd41f.

```
block_device.set_as_boot_disk()
```
